loqusdb/build_models/variant.py

```diff
@@ -190,7 +196,9 @@ def build_variant(variant, case_obj, case_id=None, gq_threshold=None, gq_qual=Fa
             ind_pos = ind_obj["ind_index"]
 
             if gq_qual:
-                gq = int(variant.QUAL)
+                gq = 0
```

Comment:
But then it seems we assumed incorrectly from the logs that the problem was with the SV file 🤔 I haven't looked at tnscope... Were you able to reproduce the error by uploading the SNVs individually?

Reply:
Cool, then it's not as bad as it seemed, thanks again for helping with this